metadata.go

```diff
@@ -216,37 +263,24 @@ func getLocalImages() (imageMap map[ImageIDType][]RepoTagType, e error) {
 	}
 
 	// make map from each imageID to all of its aliases (repo+tag)
-	imageMap = make(map[ImageIDType][]RepoTagType)
+	imageMap = make(ImageToRepoTagMap)
 	for _, localImage := range localImageList {
 		imageID := ImageIDType(localImage.ID)
-		for _, repoTag := range localImage.RepoTags {
+		for _, regRepoTag := range localImage.RepoTags {
 			// skip images with no repo:tag
-			if repoTag == "" {
-				blog.Info("Image ", imageID, " has a repoTag of empty string.")
+			if regRepoTag == "" || regRepoTag == "\u003cnone\u003e:\u003cnone\u003e" || regRepoTag == "<none>:<none>" {
+				blog.Info("Image ", imageID, " has a <none>:<none> repository:tag.")
```

Comment:
If the first argument to blog.Info has type string, it must be a printf-like format string.